specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/examples/licenseProfile/LicenseProfile_Update.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscriptionId}",
+    "resourceGroupName": "myResourceGroup",
+    "machineName": "myMachine",
+    "licenseProfileName": "default",
+    "api-version": "2023-06-20-preview",
+    "parameters": {
+      "properties": {
+        "esuProfile": {
+          "arcEsuEnrolled": "false"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
as with create/update, this will always be an async response

Reply:
ARM restriction